packages/calcite-components/src/components/sheet/sheet.stories.ts

```diff
@@ -0,0 +1,79 @@
+import { select } from "@storybook/addon-knobs";
+import { boolean, storyFilters } from "../../../.storybook/helpers";
+import { modesDarkDefault } from "../../../.storybook/utils";
+import readme from "./readme.md";
+import { html } from "../../../support/formatting";
+
+export default {
+  title: "Components/Sheet",
+  parameters: {
+    notes: readme,
+    chromatic: {
+      delay: 500,
+    },
+  },
+  ...storyFilters(),
+};
+
+const panelHTML = html`<calcite-panel heading="Ultrices neque"
+  ><p>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
+    aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+    Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur
+    sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+  </p>
+  <calcite-button slot="footer" width="half" appearance="outline">tincidunt lobortis</calcite-button>
+  <calcite-button slot="footer" width="half" appearance="outline">amet porttitor</calcite-button>
+</calcite-panel>`;
+
+export const simple = (): string =>
+  html`
+    <calcite-sheet
+      label="libero nunc"
+      ${boolean("open", true)}
+      position="${select("position", ["inline-start", "inline-end", "block-start", "block-end"], "inline-start")}"
+      display-mode="${select("display-mode", ["overlay", "float"], "overlay")}"
+      >${panelHTML}</calcite-sheet
+    >
+  `;
+
+export const simpleDarkMode = (): string =>
+  html`
+    <calcite-sheet
+      label="libero nunc"
+      ${boolean("open", true)}
+      ${select("position", ["inline-start", "inline-end", "block-start", "block-end"], "inline-start")}
+      ${select("display-mode", ["overlay", "float"], "overlay")}
+      >${panelHTML}</calcite-sheet
+    >
+  `;
+simpleDarkMode.parameters = { modes: modesDarkDefault };
+
+export const inlineStartfloat_TestOnly = (): string =>
```

Comment:
More of a curiosity - Chromatic seems to crop the screenshot so we don't actually see the `float` inset position, just the border radius. It looks correct in associated Storybook build and locally though:


![Screenshot 2023-08-28 at 9 58 56 AM](https://github.com/Esri/calcite-design-system/assets/4733155/4852fa28-2fc7-42e0-ac1d-c91e2c399c34)
![Screenshot 2023-08-28 at 9 58 51 AM](https://github.com/Esri/calcite-design-system/assets/4733155/029f827c-ca59-473e-8f3b-4ecd56b02051)

Reply:
@macandcheese could it be a browser version difference?

Reply:
Can't seem to replicate the positioning of the screenshot in FF / Safari - but we may need another rule to fix the border radius in Safari, I think maybe due to the container css changes?
![Screenshot 2023-08-28 at 10 12 22 AM](https://github.com/Esri/calcite-design-system/assets/4733155/1d7634a9-6487-4578-96c0-245eb71d0fff)

Reply:
Ill put back some host styles and see if that fixes the screenshots.

Reply:
@macandcheese I think the screens are good now. Let me know if its good to merge

Reply:
Can you kick off another Chromatic? Inset looks good locally but wasn't showing up before outside of Chromatic. I do still see the non-rounded border in Safari. Modal / Shell Panel in 'float' don't show this so maybe we just missed one of the styles?

Reply:
@macandcheese when we set the border radius, we need to set overflow:hidden so the panel doesn't go outside of where it should.